web/pom.xml

```diff
@@ -1,49 +1,40 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0"
   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
   <modelVersion>4.0.0</modelVersion>
+
   <parent>
     <groupId>com.microsoft.azure.simpletodo</groupId>
     <artifactId>simple-todo-parent</artifactId>
     <version>0.0.1-SNAPSHOT</version>
-    <relativePath>../pom.xml</relativePath>
-    <!-- lookup parent from repository -->
```

Comment:
`<relativePath>` is not necessary because it equals to the default velue. And the comment (`<!-- lookup parent from repository -->`) is not right. So, I deleted them.